src/create_torrent.cpp

```diff
@@ -48,6 +48,7 @@ POSSIBILITY OF SUCH DAMAGE.
 #include "libtorrent/aux_/directory.hpp"
 #include "libtorrent/aux_/time.hpp" // for posix_time
 #include "libtorrent/disk_interface.hpp"
+#include "libtorrent/piece_picker.hpp"
```

Comment:
It looks strange that `create_torrent` depends on `piece_picker`. It would be better to define this limit outside to both `create_torrent` and `piece_picker` classes.

Reply:
yeah, that's a good point. This should be a well defined and documented limit